File: languages/csharp_unity/Family.cs
using System;
using System.Collections.Generic;
[System.Serializable]
public class Family : BaseElement
{
    // Identity
    public string Spirit;
    public string History;
    public List<string> TraditionsIds; = new List<string>();
    public List<string> TraitsIds; = new List<string>();
    public List<string> AbilitiesIds; = new List<string>();
    public List<string> LanguagesIds; = new List<string>();
    public List<string> AncestorsIds; = new List<string>();
    // World
    public string Reputation;
    public List<string> EstatesIds; = new List<string>();
    public List<string> GovernsIds; = new List<string>();
    public List<string> HeirloomsIds; = new List<string>();
    public List<string> CreaturesIds; = new List<string>();
}

File: languages/typescript/Family.ts
import { BaseElement } from './base_elements';

export interface Family extends BaseElement {
  // Community
  spirit?: string | null;
  alliancesIds?: string[] | null;
  rivalriesIds?: string[] | null;
  // Lineage
  history?: string | null;
  ancestorsIds?: string[] | null;
  traitsIds?: string[] | null;
  abilitiesIds?: string[] | null;
  languagesIds?: string[] | null;
  // World
  status?: string | null;
  competitionIds?: string[] | null;
  administratesIds?: string[] | null;
  creaturesIds?: string[] | null;
  // Legacy
  traditions?: string | null;
  estateId?: string | null;
  heirloomsIds?: string[] | null;
}
